docs/public/architecture/dogstatsd/internals.md

```diff
@@ -1,100 +1,92 @@
-# DogStatsD
+# DogStatsD internals
 
-Information on DogStatsD, configuration and troubleshooting is available in the Datagdog documentation:
-[https://docs.datadoghq.com/developers/dogstatsd/][1]
```

Comment:
Would it make sense to keep a link to the user documentation?

Reply:
Do you think having that link would be useful to us? Customers would not be using with page.

Reply:
Not to us, but to people that land on this page from internet search. There is no indication on the page that these are internal docs not intended for end users.

Reply:
Added!